src/v/pandaproxy/schema_registry/seq_writer.cc

```diff
@@ -391,6 +468,12 @@ seq_writer::do_delete_subject_impermanent(subject sub, model::offset write_at) {
       delete_subject_key{.seq{write_at}, .node{_node_id}, .sub{sub}},
       delete_subject_value{.sub{sub}});
 
+    auto mode = co_await ss::coroutine::as_future(
+      _store.get_subject_mode_written_at(sub));
+    if (!mode.failed()) {
+        rb(mode.get());
+    }
+
```

Comment:
>is it generally advisable to explicitly get_exception (or discard or whatever) on the other side of the branch here?

Yes, you need to eat the exception out of ready future to avoid abandoned exceptional future.

same below for `conf`